.github/workflows/build-rust-binaries.yaml

```diff
@@ -64,23 +66,28 @@ jobs:
       - name: Rust cache
         uses: Swatinem/rust-cache@v2
       - name: Install zig
-        if: startsWith(matrix.target, 'x86_64-unknown-linux-musl') || startsWith(matrix.target, 'aarch64-unknown-linux-musl')
+        if: contains(matrix.target, 'linux')
         uses: mlugg/setup-zig@v2
       - name: Install zigbuild
-        if: startsWith(matrix.target, 'x86_64-unknown-linux-musl') || startsWith(matrix.target, 'aarch64-unknown-linux-musl')
+        if: contains(matrix.target, 'linux')
         run: cargo install cargo-zigbuild
       # Linux MUSL toolchain (native, not cross)
       - name: Install musl tools (Linux)
-        if: startsWith(matrix.target, 'x86_64-unknown-linux-musl') || startsWith(matrix.target, 'aarch64-unknown-linux-musl')
+        if: contains(matrix.target, 'linux')
         run: |
           sudo apt-get update
           sudo apt-get install -y musl-tools pkg-config build-essential
 
       - name: Build Rust Library (Cargo)
-        if: ${{ !(startsWith(matrix.target, 'x86_64-unknown-linux-musl') || startsWith(matrix.target, 'aarch64-unknown-linux-musl')) }}
+        if: ${{ !contains(matrix.target, 'linux') }}
         run: cargo build -p yggdrasilffi --release --target ${{ matrix.target }};
 
       - name: Build Rust library (zigbuild/musl)
-        if: startsWith(matrix.target, 'x86_64-unknown-linux-musl') || startsWith(matrix.target, 'aarch64-unknown-linux-musl')
+        if: contains(matrix.target, 'musl')
         run: cargo zigbuild -p yggdrasilffi --release --target ${{ matrix.target }};
 
+      - name: Build Rust library (zigbuild/gnu, ${{ matrix.zigTarget }})
+        if: contains(matrix.target, 'linux-gnu')
```

Comment:
Is this the same as `!contains(matrix.target, 'musl')`? Just for simplicity, when reading it's either with musl or without